js/url_search_params.ts

```diff
@@ -24,6 +24,12 @@ export class URLSearchParams {
     } else if (Array.isArray(init)) {
       // Overload: sequence<sequence<USVString>>
       for (const tuple of init) {
+        // If pair does not contain exactly two items, then throw a TypeError.
+        if (tuple.length !== 2) {
```

Comment:
Looks like I missed this check when I did the original, thanks you for fixing this!